front/src/router/index.js

```diff
@@ -9,6 +20,114 @@ const routes = [
     path: '/',
     name: 'Home',
     component: Home
+  },
+  {
+    path: '/housing',
+    name: 'Housing',
+    component: Housing
+  },
+  {
+    path: '/transport',
+    name: 'Transport',
+    component: Transport
+  },
+  {
+    path: '/leisure',
+    name: 'Leisure',
+    component: Leisure
+  },
+  {
+    path: '/food',
+    name: 'Food',
+    component: Food
+  },
+  {
+    path: '/hot-tours',
+    name: 'HotTours',
+    component: HotTours
+  },
+  {
+    path: '/articles',
+    name: 'Articles',
+    component: Articles
+  },
+  {
+    path: '/my/tickets',
```

Comment:
you can also use children path
/my
children: {
}

Reply:
added children